mongo/doc.go

```diff
@@ -102,16 +102,16 @@
 // using a different DNS server (8.8.8.8 is the common default), and, if that's not possible, avoiding the "mongodb+srv"
 // scheme.
 //
-// # Client Side Encryption
+// # Client-Side Field Level Encryption
 //
-// Client-side encryption is a new feature in MongoDB 4.2 that allows specific data fields to be encrypted. Using this
+// Client-side field level encryption is a new feature in MongoDB 4.2 that allows specific data fields to be encrypted. Using this
 // feature requires specifying the "cse" build tag during compilation:
 //
 //	go build -tags cse
 //
 // Note: Auto encryption is an enterprise- and Atlas-only feature.
 //
-// The libmongocrypt C library is required when using client-side encryption. Specific versions of libmongocrypt
+// The libmongocrypt C library is required when using client-side field level encryption. Specific versions of libmongocrypt
```

Comment:
Can we extend the doc.go file to include this information from the C PR: https://github.com/mongodb/mongo-c-driver/pull/1255/files#diff-36bfd348aee753ebb9428c3b6d64ebf7fdf3bcdccd3a8338acd480bb70a66f7bR33-R37

The AutoEncryptionOptions are here: https://github.com/mongodb/mongo-go-driver/blob/master/mongo/options/autoencryptionoptions.go